src/main/java/org/scijava/parallel/ParallelizationParadigmProfile.java

```diff
@@ -35,7 +35,7 @@ String getName() {
 	 * Gets the {@link ParallelizationParadigm} type which is to be used in this
 	 * profile
 	 * 
-	 * @return {@link Class<ParallelizationParadigm>}
+	 * @return {@link org.scijava.parallel.ParallelizationParadigm}
```

Comment:
The method does return a Class type. What was the rationale behind the performed change?

Reply:
Warnings - generics are not referred in JavaDOC because it is not part of signature. On the other hand, a type Class should be here instead.